recipes/quickfast/all/CMakeLists.txt

```diff
@@ -0,0 +1,76 @@
+# based on: https://github.com/microsoft/vcpkg/blob/master/ports/quickfast/CMakeLists.txt
```

Comment:
An alternative to having this cmake script is to:
1. make this cmake script a patch that creates a cmake script in the `source_subfolder`; and
2. create  a common cmake script here that does the ordinary include of conanbuildinfo.cmake and add_subdirectory(source_subfolder).

This allows a future cmake script for quickfast.